pkgs/os-specific/linux/system76-io/default.nix

```
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchFromGitHub, kernel }:
 let
-  version = "1.0.2";
-  sha256 = "sha256-DWUjQmoojkzFv1p4Xyt0kOwwqQ216ocO5yR/ujhhMPA=";
+  version = "1.0.4";
+  sha256 = "sha256-VE6sCehjXlRuOVcK4EN2H+FhaVaBi/jrAYx4TZjbreA=";
```

```suggestion
  hash = "sha256-XXXXX";
```
Please use `hash` instead of `sha256` and regenerate the hash.
#325892

---

Should be good now.